pyrsistent/_pclass.py

```diff
@@ -68,6 +68,10 @@ def __new__(cls, **kwargs):    # Support *args?
 
         check_global_invariants(result, cls._pclass_invariants)
 
+        try:
+            result._hash = hash(tuple((key, getattr(result, key, _MISSING_VALUE)) for key in result._pclass_fields))
+        except:
+           result._hash = None
```

Comment:
Where is _hash being used? It's a new attribute, but I don' t see it read anywhere.